Phindr3D-Python/src/Data/ImageStack.py

```diff
@@ -22,8 +24,21 @@ class ImageStack:
 
     def __init__(self):
         """ImageStack class constructor"""
+        self.stackNumber = None
+        self.layers = {}
         pass
 
+    def setStackNumber(self, number):
+        self.stackNumber = number
+
+    def addLayers(self, list):
+        for layer in list:
+            key = layer[5]
```

Comment:
Why is this assumed to be layer[5]? Can you make this more general? I think you suggested a dictionary?